lib/zipline/zip_generator.rb

```diff
@@ -59,8 +59,10 @@ def normalize(file)
       end
     end
 
-    def write_file(streamer, file, name)
-      streamer.write_deflated_file(name) do |writer_for_file|
+    def write_file(streamer, file, name, mtime)
+      mtime = mtime.nil? ? Time.now.utc : mtime
```

Comment:
what is the default behavior if you don't pass this to zip_tricks? Can we just default to that?

Reply:
It uses `Time.now.utc` as well.
https://github.com/RTrampov/zip_tricks/blob/c1fe786ad72561a3f3d924f5f659d2642bd68365/lib/zip_tricks/streamer.rb#L310

I'll try to rewrite it as you suggested.